Set change notif

vguttmann opened the pull request:
Closes #122 

## Proposed changes 

- Added notifications upon changing the set
- Added the possibility to disable said notifications
- Tested with Dualshock 3
- Commits are a bit messy, I had some fights with git.

----


<!-- 
    Please, go through these steps before you submit a PR.

    Make sure that your PR is not a duplicate.

    If not, then make sure that:

    - You have done your changes in a separate branch.

    - You have a descriptive, semantic commit messages with a short title (first line). E.g. `fix(calibration): fix calibration dialog buttons`

    - Provide a description of your changes, wih screenshots if possible

    - Put closes #XXXX in your comment to auto-close the issue that your PR fixes (if such).
    
    - When merging, don't forget to squash commits! -->

vguttmann commented:
I'll just clean it up by hand and then commit

pktiuk commented:
@vguttmann   
> I'll just clean it up by hand and then commit

Don't worry about this. That's why we can mark Pull Requests as WIP (work in progress) and push our commits with flag `--force` without any problems. There is no need for opening another pr, just push with force, and you will overwrite your previous commits.   
In case of messy commits we can just squash them during merge (merge all of them into one commit).

vguttmann commented:
Well, the problem with the first was that I had the commits actually in that file. Like multiple sections of the same code separated by commit markings and such. 

Because I did not want to do this by hand, and that I assumed my amateur git skills caused this, I just copied the files, cleaned them up, deleted my fork and readded the files. It was a lot quicker than digging through git commands. 

In the end, it all works like it should, and it's reasonably clean regarding commits
